pipeline/MUSIC_DATA.csv

```diff
@@ -0,0 +1,70 @@
+item_type,item_description,album_title,artist_name,country,amount_paid_usd,genres,release_date,sale_date,sale_year,sale_month
```

Comment:
Probably want to add *.csv to .gitignore so we don't clog up the repo with data files

Reply:
Added.